### PR TITLE
Correct GetOders 

### LINKED ISSUE #83: restore sort on order search endpoint
Turns out just because it isn't noted in the serializer doesn't mean it isn't available for use.
So sorting is an option, and according to the code the only sorting option is by date.

### PR DESCRIPTION
**Pull request by nickcoleman**
Add sort (request) and tracking_numbers object (response) definitions to GetOrders

Sort was incorrectly removed from the last update.  it has been added back.

The GetOrders response includes individual line_items which contained a list of tracking_numbers.  Each of these tracking numbers is actually an object containing information associated with each tracking number (tracking_number, shipping_carrier, shipping_method, shipping_weight, shipping_cost, shipping_date, quantity). 

fixes #83

### CODE REVIEW

File: _ordering/get_orders.md [hunk 1]
```diff
@@ -335,11 +262,34 @@ supplier_name
 : (string) The Supplier Name
 
 tracking_numbers
-: (list) The Tracking Numbers list provides a Tracking Number or list of multiple Tracking Numbers for this particular SKU.
+: (list) An Order may be brokend in to multiple shipment each of which has its own Tracking Number object.  The Tracking Numbers list provides a list of those Tracking Number objects for this particular SKU.
```
Comment (baileymatthewr) [inline _ordering/get_orders.md]: I would recommend something like the following: The Tracking Numbers array parameter includes tracking number objects. Each include a tracking number, carrier, method, quantity, etc.

Comment (nickcoleman) [reply _ordering/get_orders.md]: I bow to your infinite wisdom and knowledge on all things to do with backend orders 🤓

File: _ordering/get_orders.md [hunk 2]
```diff
@@ -16,21 +16,25 @@ right_code: |
     "start_date": "2017-07-31T06:00:00.000Z",
     "end_date": "2018-08-03T06:00:00.000Z",
     "term": "",
-    "org_uuids": []
+    "org_uuids": [],
+    "sort": {
+      "key": "date",
+      "value": "des"
```
Comment (Gidgidonihah) [inline _ordering/get_orders.md]: is it `des` or `desc`?

File: _ordering/get_orders.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: /orders/
 name: Get Orders
-position: 0
+position: 3.00
```
Comment (Gidgidonihah) [inline _ordering/get_orders.md]: Why are we making manual ordering changes?

Comment (Gidgidonihah) [reply _ordering/get_orders.md]: As you explained it on the call, it makes sense. But sure to add a note to the readme about it.

File: _ordering/get_orders.md (hunk 2 above)
Comment (nickcoleman) [reply _ordering/get_orders.md]: The code checks to see if it's 'asc'.  if "sort" not 'asc', it descending.  I'll update to make that more clear.

**Pull request merged**